lib/resources/r4/related_person_patch.yaml

```diff
@@ -128,7 +139,47 @@ operations:
       }
     note: >
       <ul>
-        <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+        <li>Requires a <a href="#test-relationship-id">test operation</a> to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+      </ul>
+
+  - name: replace-relationship-extensions
```

Comment:
nit: let's make `extensions` -> `extension` since that's the FHIR field name.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR145